package.json

```diff
@@ -5,10 +5,9 @@
     "test-scss": "sass-lint static/**/*.scss --verbose --no-exit",
     "build": "yarn run build-js && yarn run build-css",
     "build-css": "node-sass --include-path node_modules static/sass --output static/css && postcss --use autoprefixer --replace 'static/css/**/*.css' && postcss --use cssnano --dir static/minified 'static/css/**/*.css'",
-    "build-js": "yarn run copy-3rd-party-js && yarn run build-publisher && yarn run build-public",
+    "build-js": "yarn run copy-3rd-party-js && yarn run build-js-transpile",
     "copy-3rd-party-js": "cp node_modules/d3/build/d3.min.js static/js/modules && cp node_modules/d3-geo/build/d3-geo.min.js static/js/modules && cp node_modules/topojson-client/dist/topojson-client.min.js static/js/modules && cp node_modules/billboard.js/dist/billboard.min.js static/js/modules && cp node_modules/moment/min/moment.min.js static/js/modules",
-    "build-publisher": "node_modules/.bin/rollup static/js/publisher/publisher.js --o static/js/dist/publisher.js -f iife -n snapcraft.publisher",
-    "build-public": "node_modules/.bin/rollup static/js/public/public.js --o static/js/dist/public.js -f iife -n snapcraft.public",
+    "build-js-transpile": "node_modules/.bin/rollup -c",
```

Comment:
I believe from scripts in package.json simply `"rollout -c"` should work, as it should have installed modules in path for this project.

Reply:
Yeah it should, but I've had issues with build environments before where they don't properly link though I notice sass-lint is called 'normally' so I'll update. Thanks @bartaz